### ISSUE TITLE
Change Burst Criteria

### ISSUE THREAD

**Issue by giulioberna** [Mensen/swa-matlab#5]
As previously discussed, I think it would be nice to add a step for detection of potential missing waves in a burst (e.g., lowering the detection crtieria by a 10-20%) for a window comprised between the supposed beginning and end of the burst.

Looking for missing waves at the beginning and at the end of the burst (ie, before first wave and after last wave) could be a bit more risky (so I would start with missing waves inside the detected bursts limits), but maybe we should try it at some point also.

**Comment by Mensen** [comment]
I agree that searching for a true beginning might be quite difficult considering we may lower the threshold too much and it would never stop finding waves... but an absolute minimum could nonetheless be specified, rather than just repeatedly dropping the amplitude criteria by 10% after a wave is found.

So something like:
if next wave is close in time... drop amplitude criteria by 10% to a limit of 30% of the original etc

**Comment by giulioberna** [comment]
I think for most analyses, it would be good enough to find missing waves between the beginning and ending points already detected... for instance, if you want to calculate the density of waves in the burst.

The search for waves at the beginning and at the end is more risky and should probably be optional (the idea of a maximum drop in amplitude is good I think).
